<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loadRoi</class>
 <widget class="QWidget" name="loadRoi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Region of Interest (ROI)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(42, 42, 42);
}</string>
  </property>
  <widget class="QLabel" name="editImageDisplayLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-30</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 29px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Load ROI:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="newFolderPathLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>ROI file: (.pkl)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="roiPathInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(249, 249, 249);
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseFileButton">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>140</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Choose File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearFileButton">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>140</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Clear File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openRoiButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="wrongImageWarning">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: red;
	background-color: rgba(0,0,0,0);
	font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Selected ROI is for a different image!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
